### PR TITLE
Add explicit warning for hostpath-provisioner

### PR DESCRIPTION
**Pull request by svollath**

### CODE REVIEW

File: xml/cap_depl_install_minimal.xml [hunk 1]
```diff
@@ -305,6 +305,20 @@ Happy Helming!
      is an easy option for a minimal test deployment.
  </para>
 
+   <warning>
+        <title>Warning: Enable hostpath-provisioner on CaaSP</title>
```
Comment (cjschroder) [inline xml/cap_depl_install_minimal.xml]: Caasp should be &caasp;  which expands to Caas Platform.

File: xml/cap_depl_install_minimal.xml [hunk 2]
```diff
@@ -305,6 +305,20 @@ Happy Helming!
...
+        <para>
+        CaaSP is configured as a multi-node kubernetes setup with a minimum of one master and two workers.
```
Comment (cjschroder) [inline xml/cap_depl_install_minimal.xml]: &caasp; again, and Kubernetes is &kube;

File: xml/cap_depl_install_minimal.xml [hunk 3]
```diff
@@ -305,6 +305,20 @@ Happy Helming!
      is an easy option for a minimal test deployment.
  </para>
 
+   <warning>
+        <title>Warning: Enable hostpath-provisioner on CaaSP</title>
+        <para>
+        CaaSP is configured as a multi-node kubernetes setup with a minimum of one master and two workers. 
+        Enabling hostpath provisioning on CaaSP will use local storage on each of these nodes.
```
Comment (cjschroder) [inline xml/cap_depl_install_minimal.xml]: &caasp;

File: xml/cap_depl_install_minimal.xml [hunk 4]
```diff
@@ -305,6 +305,20 @@ Happy Helming!
      is an easy option for a minimal test deployment.
  </para>
 
+   <warning>
+        <title>Warning: Enable hostpath-provisioner on CaaSP</title>
+        <para>
+        CaaSP is configured as a multi-node kubernetes setup with a minimum of one master and two workers. 
+        Enabling hostpath provisioning on CaaSP will use local storage on each of these nodes. 
+        Therefore persistent data stored will also only be available locally on the respective kubernetes nodes. 
+        That directly impacts use cases, where CAP containers could restart on a different kubernetes worker
```
Comment (cjschroder) [inline xml/cap_depl_install_minimal.xml]: Replace CAP with &scf; for SUSE Cloud Foundry. See section 1.1. The product managers want us to be specific about the various components that make up the SUSE Cloud Application Platform. And to use the correct long product names.

File: xml/cap_depl_install_minimal.xml [hunk 5]
```diff
@@ -305,6 +305,20 @@ Happy Helming!
...
+        <title>Warning: Enable hostpath-provisioner on CaaSP</title>
+        <para>
+        CaaSP is configured as a multi-node kubernetes setup with a minimum of one master and two workers. 
+        Enabling hostpath provisioning on CaaSP will use local storage on each of these nodes. 
+        Therefore persistent data stored will also only be available locally on the respective kubernetes nodes. 
+        That directly impacts use cases, where CAP containers could restart on a different kubernetes worker 
+        than before. Typically that is about restarting containers for high availability or update tests. 
+        If a container starts on a different worker than before, it will miss its persistent data, leading to various other side effects. 
+        In addition, hostpath-provisioner uses the local root filesystem of the kubernetes node. If it 
+        runs out of disk space your kubernetes node won't work anymore.
```
Comment (cjschroder) [inline xml/cap_depl_install_minimal.xml]: &kube; x 2 :)

Comment (cjschroder) [reply xml/cap_depl_install_minimal.xml]: Please wrap lines at 80 characters.
Thanks for this!

File: xml/cap_depl_install_minimal.xml (hunk 1 above)
Comment (cjschroder) [reply xml/cap_depl_install_minimal.xml]: This should probably be &scf; not &caasp;

Comment (cjschroder) [reply xml/cap_depl_install_minimal.xml]: No it should be &caasp; :)

File: xml/cap_depl_install_minimal.xml (hunk 5 above)
Comment (cjschroder) [reply xml/cap_depl_install_minimal.xml]: If you branch from Develop then I can make markup corrections.

**Comment by svollath** [comment]
Sorry, I exactly looked up the existing warning before, where CaaSP is plain text, so you might want to change that (within the warning "Initialize Only the &helm; Client").
For &scf; the build failed, so I think that's &cap;.
... should be fine now.
THX

**Comment by cjschroder** [comment]
I need more coffee...the Cloud Foundry entities are &cf; and &susecf;. Entities are defined in entity-decl.ent. Thank you!

**Pull request merged**